crates/egui/src/widgets/button.rs

```diff
@@ -246,18 +245,27 @@ impl Widget for Button {
 /// ui.checkbox(&mut my_bool, "Checked");
 /// ui.add(egui::Checkbox::new(&mut my_bool, "Checked"));
 /// # });
+/// // You can also create a checkbox without a text label
+/// ui.add(egui::Checkbox::without_text(&mut my_bool));
 /// ```
 #[must_use = "You should put this widget in an ui with `ui.add(widget);`"]
 pub struct Checkbox<'a> {
     checked: &'a mut bool,
-    text: WidgetText,
+    text: Option<WidgetText>,
 }
 
 impl<'a> Checkbox<'a> {
     pub fn new(checked: &'a mut bool, text: impl Into<WidgetText>) -> Self {
         Checkbox {
             checked,
-            text: text.into(),
+            text: Some(text.into()),
+        }
+    }
+
+    pub fn without_text(checked: &'a mut bool) -> Self {
+        Checkbox {
+            checked,
+            text: None,
```

Comment:
```suggestion
    pub fn without_text(checked: &'a mut bool) -> Self {
        Self::new(checked, WidgetText::default())
```

and then you won't need any other change